assets/airports/panc.json

```diff
@@ -0,0 +1,2164 @@
+                "25L": [],
+                "25R": [],
+                "33": []
+            },
+            "draw": [
+                ["JOH", "TAZLI", "ELLAM"]
+            ]
+        },
+        "KROTO3": {
+            "icao": "KROTO3",
+            "name": "Kroto Three",
+            "entryPoints": {
+                "GKN": ["GKN", "IRONC", "TKA"],
+                "KKYLE": ["KKYLE", "AKMAX", "ROHHN"],
+                "PUYVO": ["PUYVO", "AILEE", "TKA"],
+                "WINOR": ["WINOR", "FFITZ", "ROHHN"]
+            },
+            "body": [["KROTO", "A90+|S250"]],
+            "rwy": {
+                "7L": [["NHCOL", "A50|S240-"], "ASLEE", ["CULEN", "A40"], "#252"],
+                "7R": [["NHCOL", "A50|S240-"], "ASLEE", ["CULEN", "A40"], "#252"],
+                "15": [["TOMMM", "A50+"], ["CARDD", "A30"]]
+            },
+            "draw": [
+                ["GKN", "IRONC", "TKA", "KROTO", "NHCOL", "ASLEE", "CULEN"],
+                ["KKYLE", "AKMAX", "ROHHN", "KROTO", "TOMMM", "CARDD"],
+                ["WINOR", "FFITZ", "ROHHN"],
+                ["PUYVO", "AILEE", "TKA"]
+            ]
+        },
+        "NEELL5": {
+            "icao": "NEELL5",
+            "name": "Neil Five",
+            "entryPoints": {
+                "AKGAS": [["AKGAS", "A250+"], ["GEDEE", "A160+"]],
+                "BAITT": [["BAITT", "A250+"], ["LEWYY", "A160+"]],
+                "HAMND": [["HAMND", "A250+"], ["BTWEN", "A210+"], ["CKING", "A160+"]]
+            },
+            "body": [["SNAAG", "A100+"], "NEELL", ["SHNKN", "A30+"]],
+            "rwy": {
+                "7L": [["ELIAS", "A20+"]],
+                "7R": [["AINKK", "A20+"]]
+            },
+            "draw": [
+                ["AKGAS", "GEDEE", "SNAAG", "NEELL", "SHNKN", "ELIAS"],
+                ["HAMND", "BTWEN", "CKING", "SNAAG"],
+                ["BAITT", "LEWYY", "SNAAG"],
+                ["SHNKN", "AINKK"]
+            ]
+        },
+        "PTERS2": {
+            "icao": "PTERS2",
+            "name": "Peters Two",
+            "entryPoints": {
+                "GKN": [["GKN", "A260+"]],
+                "JOH": ["JOH", "DENSN"],
+                "MDO": [["MDO", "A260+"], "DENSN"],
+                "REMBY": [["REMBY", "A260+"]],
+                "TIEKL": [["TIEKL", "A260+"]],
+                "TMSON": [["TMSON", "A260+"]]
+            },
+            "body": [["PTERS", "A160+"], ["SKENS", "A130+|A150-"], ["MMMAT", "A100+"], ["BBELL", "A80+"], ["DDOUG", "A80"], "LMPEE", "#327"],
+            "rwy": {
+                "15": []
+            },
+            "draw": [
+                ["GKN", "PTERS", "SKENS", "MMMAT", "BBELL", "DDOUG", "LMPEE"],
+                ["JOH", "DENSN", "PTERS"],
+                ["REMBY", "PTERS"],
+                ["TIEKL", "PTERS"],
+                ["TMSON", "PTERS"],
+                ["MDO", "DENSN"]
+            ]
+        },
+        "TAGER6": {
+            "icao": "TAGER6",
+            "name": "Tagger Six",
```

Comment:
This is now the `TAGER8`. Just a name change needed, the content is still accurate.

Reply:
It wasn't renamed, I just numbered it incorrectly :man_facepalming:

Reply:
done 👍